problems/4.arrays.js

```diff
@@ -12,18 +19,28 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+const duplicateArray = (arr) => {
+  let dupArray = []
+  for(let i=0; i < arr.length; i++){
+    dupArray.push(arr[i])
+  }
+  return [...arr,...dupArray]
 }
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
-
+const removeFirstThreeElements = (arr) => {
+  if(arr.length < 4){
+    arr.pop()
+    arr.shift
```

Comment:
These two lines aren't working in the way you want. The .shfit should be invoked though, but also shouldn't be here.